modules/microshift-certificate-lifetime.adoc

```diff
@@ -48,4 +48,4 @@ The following situations describe {microshift-short} actions during long-term ce
 .. When a long-term certificate is 8.5 to 9 years old, it is rotated when {microshift-short} starts or restarts.
 
 . Automatic restart for rotation:
-.. When a long-term certificate is more than 9 years old, {microshift-short} can automatically restart to rotate and apply a new certificate.
+.. When a long-term certificate is more than 9 years old, {microshift-short} can automatically restart to rotate and apply a new certificate.
```

Comment:
```suggestion
.. When a long-term certificate is more than 9 years old, {microshift-short} might automatically restart so that it can rotate and apply a new certificate.
```